packages/discovery-provider/integration_tests/tasks/entity_manager/test_playlist_entity_manager.py

```diff
@@ -594,7 +594,7 @@ def get_events_side_effect(_, tx_receipt):
         )
         assert len(playlists_1) == 1
         playlist_1 = playlists_1[0]
-        assert datetime.timestamp(playlist_1.last_added_to) == 1585336422
+        assert datetime.timestamp(playlist_1.last_added_to) == 1660927554
```

Comment:
these are changed bc now we're updating to the last added track's metadata_time even in playlist entity manager create. previously we were only doing so on playlist update.